test/serializers/attribute_test.rb

```diff
@@ -81,6 +81,23 @@ def id
         assert_equal('custom', hash[:blog][:id])
       end
 
+      def test_id_with_question_mark_has_question_mark_removed
+        serializer = Class.new(ActiveModel::Serializer) do
+          # TODO: if we were to remove the question mark automatically,
+          #       what do we do if we want to call a method that has a
+          #       question mark.
+          #       e.g.:
+          #               def my_question?; ....; end
```

Comment:
Still is a good test to have to demonstrate behavior.  Need to confirm json api adapter strips it